cli/cmd/env/configure.go

```diff
@@ -129,7 +124,6 @@ func ConfigureCmd(ch *cmdutil.Helper) *cobra.Command {
 
 	configureCommand.Flags().SortFlags = false
 	configureCommand.Flags().StringVar(&projectPath, "path", ".", "Project directory")
-	configureCommand.Flags().StringVar(&subPath, "subpath", "", "Project path to sub directory of a larger repository")
```

Comment:
Yes, but don't want to do it now because for `rill deploy`, it actually needs to *know* what the subpath is, but I couldn't find a way to get that info from `go-git`.